web/src/app/components/loi-panel-header/loi-panel-header.component.ts

```diff
@@ -133,13 +128,25 @@ export class LocationOfInterestPanelHeaderComponent
     });
   }
 
+  /** A label for a given geometry type. Defaults to 'Polygon'. */
+  private geometryTypeLabel(geometryType?: GeometryType): string {
```

Comment:
I saw you added a similar function in `geometry.ts`, do you want to refer to that instead?

Reply:
I actually moved that back in here, per a request from Gino.